index.d.ts

```diff
@@ -403,18 +403,15 @@ declare module 'fastest-validator' {
 	 * Validation schema definition for custom inline validator
 	 * @see https://github.com/icebob/fastest-validator#custom-validator
 	 */
-	interface RuleCustomInline<T = any> extends RuleCustom {
```

Comment:
Not sure that we should remove generic form here. It should help detect errors in the pre-compile stage of a project.

Reply:
I think, generic type it's way to possible errors. `unknown` makes us to check all possible validation cases. But generic type promise correct input type - it's wrong. Originally input value is unknown.

Reply:
Ok. But we should keep in mind that is a breaking change and will make users update code after the update

Reply:
Can we solve that it doesn't cause breaking change? I don't plan to release a new major version.

Reply:
@icebob Generic types for RuleCustomInline and CheckerFunction was returned

Reply:
Thanks!